src/app/content/components/NudgeStudyTools/index.spec.tsx

```diff
@@ -112,6 +110,30 @@ describe('NudgeStudyTools', () => {
     expect(() => component.root.findByType(NudgeContentWrapper)).not.toThrow();
     expect(() => component.root.findByType(NudgeBackground)).not.toThrow();
     expect(() => component.root.findByType(NudgeSpotlight)).not.toThrow();
+    expect(() => component.root.findByProps({
+      id: 'i18n:nudge:study-tools:aria-label:only-highlighting',
+    })).not.toThrow();
+    expect(() => component.root.findByProps({
+      id: 'i18n:nudge:study-tools:text:only-highlighting',
+    })).not.toThrow();
+
+    jest.spyOn(studyGuidesSelect, 'hasStudyGuides')
+      .mockReturnValue(true);
+
+    component.update(<Provider store={store}>
+      <Services.Provider value={services}>
+        <MessageProvider>
+          <NudgeStudyTools/>
+        </MessageProvider>
+      </Services.Provider>
+    </Provider>);
+
+    expect(() => component.root.findByProps({
+      id: 'i18n:nudge:study-tools:aria-label:with-study-guides',
+    })).not.toThrow();
+    expect(() => component.root.findByProps({
+      id: 'i18n:nudge:study-tools:text:with-study-guides',
+    })).not.toThrow();
```

Comment:
I think it would be nice if this also asserted that the previous text is no longer shown, probably just one of the assertions from above is enough.